packages/next/image-types/global.d.ts

```diff
@@ -1,15 +1,8 @@
 // this file is conditionally added/removed to next-env.d.ts
 // if the static image import handling is enabled
 
-interface StaticImageData {
-  src: string
-  height: number
-  width: number
-  blurDataURL?: string
-}
-
 declare module '*.png' {
-  const content: StaticImageData
+  const content: import('../dist/client/image').StaticImageData
```

Comment:
TIL if I add a root import in this file TS will no longer treat it as a global declaration file, but using import within the ambient declaration works